### DISCUSSION TITLE
Não estou conseguindo entrar na app Tik Tok Company, está dando erro 451 , me ajuda

### DISCUSSION THREAD

**Discussion by renatoandre1175** [vercel/community#1151] [Help] [answered]
Não estou conseguindo entrar no app Tik Tok Company, está dando erro 451

**Comment by leiaribeiro** [comment]
> Não estou conseguindo entrar no app Tik Tok Company, está dando erro 451

**Reply by mary19-hub** [reply]
Não estou conseguindo também entrar faz dias! Oque faço? Alguém me ajuda?

**Reply by mary19-hub** [reply]
> Não estou conseguindo  também entrar faz dias! Oque faço? Alguém me ajuda?

**Reply by Tayanedias** [reply]
Também não estou conseguindo entrar!!
A menina do suporte no WhatsApp não responde

**Comment by leiaribeiro** [comment]
Não estou conseguindo entrar no app tiktok-pay

**Comment by renatoandre1175** [comment]
Estou precisando do suporte e não consigo, me ajuda

Em qua, 14 de dez de 2022 09:49, mary19-hub ***@***.***>
escreveu:

**Comment by NilmaM** [comment]
Não estou conseguindo entrar no tiktok Company

**Comment by renatoandre1175** [comment]
Em qua, 14 de dez de 2022 10:28, NilmaM ***@***.***> escreveu:

**Comment by Batista223** [comment]
Cadê ninguém da um suporte precisamos de ajuda.

**Comment by amyegan** [comment, marked as answer]
👋 Hello everyone. I believe you are all talking about a site that is hosted by Vercel, but is not owned by Vercel. You're probably seeing a generic error message that doesn't seem very helpful to you. 😥 

There isn't much we can do at the moment because we are not able to make changes to the site ourselves. You should to reach out to the site owner for help. They can work directly with our team for any support they need to get the site back online. 

I know this is a frustrating problem and I appreciate all of your patience waiting for it to be solved 🙇

**Reply by amyegan** [reply]
Update: the sites are back online now! If you have any further problems, please contact the site owner for help 🙏

**Reply by leiaribeiro** [reply]
Eu não estou conseguindo entrar no app tiktok-pay. Tá dando erro

**Reply by Julinhahhu** [reply]
N estou conseguindo  entrar no Tiktokpay estar dando erro.
